docs/iris/src/whatsnew/2.0a0.rst

```diff
@@ -15,18 +15,20 @@ Iris 2.0a0 Features
 
 .. admonition:: Showcase Feature - Dask Integration
 
-  The use of `biggus`_ to provide support for *virtual array* and
+  The use of `biggus`_ to provide support for *virtual arrays* and
   *lazy evaluation* within iris has been replaced with `dask`_.
 
-  This is a major feature enhancement, which extends the concept of *lazy data*
-  available in a :class:`~iris.cube.Cube` with a `dask Array`_ to an
-  :class:`~iris.coords.AuxCoord`, :class:`~iris.coords.CellMeasure` and
-  :class:`~iris.aux_factory.AuxCoordFactory`. Building upon `dask`_ allows
-  iris to easily leverage its rich functionality and capabilities. Most
-  notably, iris can employ either of its *threaded*, *multi-processor* or
-  *distributed* `schedulers`_ in order to fully exploit the available compute
-  resource. Also, you can use `dask Bags`_ or `dask Delayed`_ to easily
-  parallelise your algorithms.
+  In addition the concept of *lazy data*, already used for the
+  :class:`~iris.cube.Cube` data component, has now been extended to the data arrays in
+  :class:`~iris.coords.Coord` s and :class:`~iris.aux_factory.AuxCoordFactory` s.
```

Comment:
@pp-mo Can we drop the use of pluralising the classes, it just doesn't look right